simulator/pkg/debuggablescheduler/command.go

```diff
@@ -15,6 +16,11 @@ func NewSchedulerCommand(opts ...Option) (*cobra.Command, func(), error) {
 	for _, o := range opts {
 		o(opt)
 	}
+
```

Comment:
simulatorschedulerconfig.SetOutOfTreeRegistries in CreateOptionsForPlugins func no longer needed, right?.

Reply:
> simulatorschedulerconfig.SetOutOfTreeRegistries in CreateOptionsForPlugins func no longer needed, right?.

@sanposhiho Yes do I need to delete it?

Reply:
@sanposhiho All adjustments have been completed

Reply:
@sanposhiho Are there any plans to promote this MR today?